### ISSUE TITLE
Error: Metadata version mismatch for module

### ISSUE THREAD

**Issue by tbatsenko** [basst314/ngx-webcam#71]
Hi, 

I'm experiencing the same issue.
version of ngx-webcam: ^0.2.6

```
    "@angular/animations": "^4.2.4",
    "@angular/cdk": "^2.0.0-beta.12",
    "@angular/common": "^4.2.4",
    "@angular/compiler": "^4.2.4",
    "@angular/core": "^4.2.4",
    "@angular/cli": "1.4.10",
```

 exact error message:
```
Metadata version mismatch for module /.../myapp/node_modules/ngx-webcam/ngx-webcam.d.ts, found version 4, expected 3, resolving symbol AppModule in /.../myapp/src/app/app.module.ts, resolving symbol AppModule in /../myapp/src/app/app.module.ts
```
 
The minimal example to reproduce: 
- install @angular/cli": "1.4.10",
- create a new app with ng new myapp
- insert import WebcamModule to your NgModule file
- insert <webcam> tag somewhere in your html
- run ng serve, if runs ok, terminate and try to run again

_Originally posted by @tbatsenko in https://github.com/basst314/ngx-webcam/issues/39#issuecomment-595158003_

**Comment by basst314** [comment]
Hi @tbatsenko,

I released a new version of the library (0.3.0) that bumps Typescript to >=3.7 (Angular 9).
Please update your App to the latest Angular version and try with the latest version of this library.

Thanks

**Comment by basst314** [comment]
Closing this issue as it should be fixed by latest release.
